tests/test_cms_menus.py

```diff
@@ -82,33 +82,73 @@ def test_get_nodes(self):
             attr={"link_target": child2.link_target},
         )
 
-    def test_get_roots_when_draft_mode_isnt_active(self):
+    def test_get_nodes_for_versioning(self):
```

Comment:
I think it's nicer if these tests are named `get_nodes_for_versioning_enabled` and `get_nodes_for_versioning_disabled`. Will make things clearer if one sees one of these fail in the output etc.